### ISSUE TITLE
`SELECT SUM(*)` does unwanted type cast

### ISSUE THREAD

**Issue by SBrandeis** [influxdata/influxdb-python#715]
### Software versions
Influxdb version: `1.7.6`
influxdb-python version: `5.2.2`

### Steps to reproduce:
1. Create a measurement named `my_measurement` in influxdb with following points:
```
time         data_type   value
----------   ----------  -----
15578637...   Num         12.5
15578637...   Num         12.5
```

2. Run the following code:
```
>>> from influxdb import InfluxDBClient

>>> c = InfluxDBClient(...)

>>> result_set = c.query('select * from my_measurement')
... for _, generator in result_set.items():
...     for point in generator:
...         print(type(point['value']))
<class 'float'>
<class 'float'>

>>> result_set = c.query('select sum(*) from my_measurement')
... for _, generator in result_set.items():
...     for point in generator:
...         print(type(point['sum_value']))
<class 'int'>
```

The casting into 'int' type is incorrect since we are summing floats, result should still be a float.

**Comment by lovasoa** [comment]
See the issue in influxsb itself: https://github.com/influxdata/influxdb/issues/8707 
The solution they recommend is using messagepack, which I implemented in #734.

**Comment by aviau** [comment]
this wasn't an influxdb-python issue anyways afaik
